pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -150,10 +150,36 @@ func validateRule(rule *admissionregistration.Rule, fldPath *field.Path, allowSu
 	return allErrors
 }
 
+func isAcceptedAdmissionReviewVersion(v string) bool {
+	return "v1beta1" == v
+}
+
+func validateAdmissionReviewVersions(versions []string, fldPath *field.Path) field.ErrorList {
+	allErrors := field.ErrorList{}
+
+	// Currently only v1beta1 accepted in AdmissionReviewVersions
+	if len(versions) < 1 {
+		allErrors = append(allErrors, field.Required(fldPath, ""))
+	} else {
+		seen := map[string]bool{}
+		for i, v := range versions {
+			if !isAcceptedAdmissionReviewVersion(v) {
```

Comment:
we don't want to prevent versions we don't understand, or that will block a webhook that supports v1beta1 and v1 from registering itself